src/devtools/client/debugger/src/reducers/types.ts

```diff
@@ -195,9 +172,9 @@ export type BreakpointResult = {
  * @static
  */
 export type PendingBreakpoint = {
-  readonly location: PendingLocation;
+  readonly location: Location & { sourceUrl?: string };
```

Comment:
This type gets used enough that I thought about pulling out a name for it, but I decided against it for now.